snippets/multisigning.py

```diff
@@ -0,0 +1,102 @@
+"""Example of how we can multisign a transaction"""
+from xrpl.clients import JsonRpcClient
+from xrpl.core.binarycodec import encode_for_multisigning
+from xrpl.core.keypairs import sign
+from xrpl.models.requests import SubmitMultisigned
+from xrpl.models.transactions import AccountSet, Signer, SignerEntry, SignerListSet
+from xrpl.transaction import (
+    autofill,
+    safe_sign_and_autofill_transaction,
+    send_reliable_submission,
+)
+from xrpl.utils import str_to_hex
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
+
+wallet1 = generate_faucet_wallet(client, debug=True)
+wallet2 = generate_faucet_wallet(client, debug=True)
+wallet_master = generate_faucet_wallet(client, debug=True)
```

Comment:
nit: I'd rename `wallet1`/`wallet2` to indicate they're the signer wallets so later on the reader doesn't accidentally think `wallet1` is a random wallet, or the main wallet. Also, I'd recommend instantiating `wallet_master` first since it's the main wallet we need to think about. 
```suggestion
wallet_master = generate_faucet_wallet(client, debug=True)
signer_wallet_1 = generate_faucet_wallet(client, debug=True)
signer_wallet_2 = generate_faucet_wallet(client, debug=True)
```